package.json

```diff
@@ -34,8 +34,17 @@
     "@storybook/addons": "^5.1.8",
     "@storybook/components": "^5.1.10",
     "@storybook/react": "^5.1.11",
+    "@typescript-eslint/eslint-plugin": "1.x",
+    "@typescript-eslint/parser": "1.x",
     "babel-eslint": "10.x",
     "babel-loader": "^8.0.6",
+    "eslint": "6.x",
```

Comment:
This should not be in the dependency list, it is satisfied through `d2-style` and we do not want possible collisions.